constants/timeouts.go

```diff
@@ -0,0 +1,6 @@
+package constants
+
+import "time"
+
+const HTTPTimeout = 3 * time.Minute
+const DialTimeout = 10 * time.Second
```

Comment:
As these constants are some defaults for the `httpclient` package, they'd fit better in the package itself.

Reply:
We can't put them there to prevent circular dependecy
```go
constants/timeouts.go: const HTTPTimeout = 3 * time.Minute
pkg/config/cluster.go: return constants.HTTPTimeout
pkg/httpclient/client.go: Timeout: constants.HTTPTimeout,
pkg/setup/setup.go: cluster.SetTimeout(constants.HTTPTimeout)

```